pkg/apis/openebs.io/v1alpha1/custom_resource.go

```diff
@@ -231,7 +231,7 @@ type CStorPoolPhase string
 // Status written onto CStorPool and CStorVolumeReplica objects.
 const (
 	// CStorPoolStatusInit ensures the create operation is to be done, if import fails.
-	CStorPoolStatusInit CStorPoolPhase = "init"
+	CStorPoolStatusInit CStorPoolPhase = ""
```

Comment:
Its important to have init status still. The controller should initialise the status as init for a new resource when the add event is received.

Reply:
on the yaml file, it is not required to have string "init". It is changed to "". Even if status is not present, it will be taken as "".

Reply:
we are not using the status string "init" anywhere. When the request/operation is completed, we are updating the status to "online"/"offline"/etc. Can you please confirm, if we need to update "init" status also?

Reply:
As discussed, Workflow will be, watch for empty status, update status to "init", then proceed to create pool/vol, update status to "online/offline", etc